src/alttxt/phrases.py

```diff
@@ -37,8 +37,12 @@
 
     },
     # L3 note: observe which sets are not present in many large intersections
-    "AltText": "This is an UpSet plot which shows set intersection of {{visible_set_count}} sets out of {{set_count}} sets"
-    " and the largest intersection is {{max_intersection_name}} ({{max_intersection_size}}). The plot is sorted by {{sort_type}} and {{visible_non_empty_intersect_count}} non-empty intersections are shown.",
+    "AltText": "This is an UpSet plot which shows set intersection of {{visible_set_count}} sets."
```

Comment:
This sentence feels grammatically incorrect. "... which shows set intersections", or "which shows the intersections of ... sets." is probably better

Reply:
Also, it looked to me like this output "Action, and Adventure", but it should only be one set? This is a set intersection. Maybe I read it wrong.